.github/workflows/no-response.yml

```diff
@@ -18,13 +18,15 @@ jobs:
       contents: read
       issues: write
     steps:
-      - uses: lee-dohm/no-response@9bb0a4b5e6a45046f00353d5de7d90fb8bd773bb # v0.5.0
+      - uses: MBilalShafi/no-response-add-label@9657b5bc1e09373e571df54429c3c167edaad556
```

Comment:
Should we use a "general" user for GitHub actions?

Reply:
Not sure what you mean by `general` user. The action needed to be modified, so I forked the base one and published an updated one with a slightly different name.
Should we move it under https://github.com/mui ?

Reply:
I don't see a difference between it being on an external user's profile or a maintainer's profile. External user more dangerous than maintainer though (git repos are not immutable, so in theory it would be possible to craft a commit that has the same hash but different malicious code). But I'm fine with this as it is.

Alternatively I also like what we do in other scripts with `actions/github-script` and inline code in the yaml action definition, keeps things tidy: https://github.com/mui/mui-x/blob/2e50f761a0325b95d33d52faad0b6d2fccb21e8a/.github/workflows/issue-cleanup.yml#L16-L52

Reply:
No let's not waste time no that, it's not worth it. But for from-scratch actions it's probably better to have it inline.